app/javascript/chat/actions/actions.js

```diff
@@ -56,20 +57,30 @@ export function editMessage(editedMessage, successCb, failureCb) {
     .catch(failureCb);
 }
 
-export function sendOpen(activeChannelId, successCb, failureCb) {
-  fetch(`/chat_channels/${activeChannelId}/open`, {
+// export function sendOpen(activeChannelId, successCb, failureCb) {
+//   fetch(`/chat_channels/${activeChannelId}/open`, {
+//     method: 'POST',
+//     headers: {
+//       Accept: 'application/json',
+//       'X-CSRF-Token': window.csrfToken,
+//       'Content-Type': 'application/json',
+//     },
+//     body: JSON.stringify({}),
+//     credentials: 'same-origin',
+//   })
+//     .then((response) => response.json())
+//     .then(successCb)
+//     .catch(failureCb);
+// }
+
+export async function sendOpen(activeChannelId) {
```

Comment:
refactor (non-blocking): It's not clear to me what `sendOpen` means. Consider a more descriptive name explaining the intent of the function.